pages/blueprint/index.js

```diff
@@ -313,52 +339,33 @@ class BlueprintPage extends React.Component {
         <Tabs id="blueprint-tabs">
           <Tab eventKey="details" title="Details">
             <div className="tab-container row">
-              <div className="col-sm-6 col-lg-4">
-                <dl className="dl-horizontal mt-">
-                  <dt>
-                    <FormattedMessage defaultMessage="Name" />
-                  </dt>
-                  <dd>{blueprint.name}</dd>
-                  <dt>
-                    <FormattedMessage defaultMessage="Description" />
-                  </dt>
-                  {(editDescriptionVisible && (
-                    <dd>
-                      <div className="input-group">
-                        <input
-                          type="text"
-                          className="form-control"
-                          value={editDescriptionValue}
-                          onChange={this.handleChangeDescription}
-                          onKeyPress={e => this.handleEnterKey(e, "commit")}
-                        />
-                        <span className="input-group-btn">
-                          <button
-                            className="btn btn-link"
-                            type="button"
-                            onClick={() => this.handleEditDescription("commit")}
-                          >
-                            <span className="fa fa-check" />
-                          </button>
-                          <button
-                            className="btn btn-link"
-                            type="button"
-                            onClick={() => this.handleEditDescription("cancel")}
-                          >
-                            <span className="pficon pficon-close" />
-                          </button>
-                        </span>
-                      </div>
-                    </dd>
-                  )) || (
-                    <dd onClick={() => this.handleEditDescription()} role="presentation">
-                      {blueprint.description}
-                      <button className="btn btn-link" type="button">
-                        <span className="pficon pficon-edit" />
-                      </button>
-                    </dd>
-                  )}
-                </dl>
+              <div className="col-sm-12">
+                <div className="form-horizontal">
+                  <form className="form-group" onSubmit={() => this.handleEditDescription("commit")}>
```

Comment:
It'll be convenient to add `data-form="description"` here. :+1: 
There're two same forms, same edit icon, same text input. So It's hard to find description one.